sections/02_intro.md

```diff
@@ -43,7 +42,7 @@ another set of weights to produce the output `TODO: we probably need a figure
 here - I see no way that we don't include this type of description in our paper,
 despite the fact that it's been done tons of times before. - I'm really partial
 to this nature review's explanation about making non-linear problems linear -
-figure 1 [@doi:10.1038/nature14539]` These neural networks are trained by
+figure 1 [@doi:10.1038/nature14539]`. These neural networks are trained by
```

Comment:
@swamidass has changes to this section pending review in #246.  It might make the merge easier if you revert the "What is deep learning?" sub-section here and move to his pull request to edit those paragraphs.  Or we can manually merge it later.